smoke-test/run-quickstart.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -euxo pipefail
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+cd "$DIR"
+
+python3 -m venv venv
+source venv/bin/activate
+pip install --upgrade pip wheel setuptools
+pip install -r requirements.txt
+
+mkdir -p ~/.datahub/plugins/frontend/auth/
+echo "test_user:test_pass" >> ~/.datahub/plugins/frontend/auth/user.props
+
+echo "DATAHUB_VERSION = $DATAHUB_VERSION"
+DATAHUB_TELEMETRY_ENABLED=false  \
+DOCKER_COMPOSE_BASE="file://$( dirname "$DIR" )" \
+
+datahub docker quickstart --standalone_consumers --dump-logs-on-failure
```

Comment:
So by starting the image fetch process earlier in the actions, the idea is that this command will be faster?

Reply:
Yes down from 6.5 mins to 4.5 mins now. 2 mins saved here.

Reply:
While other images are being download one-by-one docker can pull some in parallel.

Reply:
Is there any way we could use github actions caches to speed this stuff up?

I know docker has some integration with github actions but haven't looked into it a ton https://docs.docker.com/build/cache/backends/gha/

Reply:
David is doing some work on this I think. So don't want to get into merge conflicts for now.